<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1540</width>
    <height>934</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 13pt &quot;URW Gothic&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ClearGraph">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>800</y>
      <width>431</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Очистить граф. поле</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TimeCompare">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>800</y>
      <width>461</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Сравнение времени</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>160</y>
      <width>604</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="Algs">
     <item row="0" column="0">
      <widget class="QRadioButton" name="CanonAlg">
       <property name="text">
        <string>Каноническое уравнение</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="BrezAlg">
       <property name="text">
        <string>Алгоритм Брезенхема</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="ParamAlg">
       <property name="text">
        <string>Параметрическое уравнение</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="MidpointAlg">
       <property name="text">
        <string>Алгоритм средней точки</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="LibAlg">
       <property name="text">
        <string>Библ. функция</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="GraphField" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>900</width>
      <height>781</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>0</y>
      <width>601</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Выбранная фигура:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>120</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Алгоритм построения:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>310</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="Colors">
     <item row="0" column="0">
      <widget class="QRadioButton" name="BlueColor">
       <property name="text">
        <string>Синий</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="PurpleColor">
       <property name="text">
        <string>Фиолетовый</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="RedColor">
       <property name="text">
        <string>Красный</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="WhiteColor">
       <property name="text">
        <string>Белый</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="GreenColor">
       <property name="text">
        <string>Зеленый</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="YellowColor">
       <property name="text">
        <string>Желтый</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>310</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="Colors_2">
     <item row="0" column="0">
      <widget class="QRadioButton" name="BlueColor_2">
       <property name="text">
        <string>Синий</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="PurpleColor_2">
       <property name="text">
        <string>Фиолетовый</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="RedColor_2">
       <property name="text">
        <string>Красный</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="WhiteColor_2">
       <property name="text">
        <string>Белый</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="GreenColor_2">
       <property name="text">
        <string>Зеленый</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="YellowColor_2">
       <property name="text">
        <string>Желтый</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>260</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Цвет фигуры:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>260</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Цвет фона:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>60</y>
      <width>604</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="Algs_2">
     <item row="0" column="1">
      <widget class="QRadioButton" name="EllipseBtn">
       <property name="text">
        <string>Эллипс</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="CircleBtn">
       <property name="text">
        <string>Окружность</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="FigureParams">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>510</y>
      <width>601</width>
      <height>101</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="DrawCircleBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>601</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Построить окружность</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="RadiusField">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>601</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Радиус</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="DrawEllipseBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>601</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Построить эллипс</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="RxField">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Ширина</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="RyField">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Высота</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>420</y>
      <width>601</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Построение окружности/эллипса:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="CenterY">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>470</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>Y центра</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="CenterX">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>470</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>X центра</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>630</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Построение спектра:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="SpectrumParams">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>680</y>
      <width>601</width>
      <height>201</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_5">
     <widget class="QPushButton" name="DrawCircleSpectrumBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>601</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Построить спектр окружностей</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="StartRField">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Начальный радиус</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="EndRField">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>0</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Конечный радиус</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="StepRField">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>70</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Шаг построения</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="CountCurclesField">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Кол-во окружностей</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QPlainTextEdit" name="StartRyField">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>0</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Начальная высота</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="CountEllipsesField">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Кол-во эллипсов</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="StepRyField">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>50</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Шаг построения по Y</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DrawEllipseSpectrumBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>601</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Построить спектр эллипсов</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="StartRxField">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Начальная ширина</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="StepRxField">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(243, 243, 243);
font: 63 14pt &quot;URW Gothic&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Шаг построения по X</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1540</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Выход</string>
    </property>
    <addaction name="Quit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Информация</string>
    </property>
    <addaction name="AuthorInfo"/>
    <addaction name="TaskInfo"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="ClearAll">
   <property name="text">
    <string>Очистить все поля</string>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman</family>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="Quit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="ClearAll_2">
   <property name="text">
    <string>Очистить все</string>
   </property>
  </action>
  <action name="Finding">
   <property name="text">
    <string>Найти четырехугольники</string>
   </property>
  </action>
  <action name="AuthorInfo">
   <property name="text">
    <string>Об авторе</string>
   </property>
  </action>
  <action name="TaskInfo">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
